quake1_example/class/_func.gd

```diff
@@ -7,8 +7,16 @@ func _get_properties(dict : Dictionary) : props = dict
 var aabb : AABB
 
 func _gen_aabb() :
-	#aabb.position = global_position
 	for m in get_children() :
 		if m is GeometryInstance3D :
 			aabb = aabb.merge(m.get_aabb())
 	aabb.position += global_position
+
+func _can_create_trigger() -> bool : return false;
```

Comment:
This flag is mostly to get around secret doors that should not generate auto-triggers, but maybe a symptom of putting too much functionality into shared methods